src/test/java/HT_tests/DeliverToTest.java

```diff
@@ -17,44 +19,124 @@
 public class DeliverToTest {
 
     private WebDriver driver;
+    private WebDriverWait webDriverWait;
 
     @BeforeTest
     public void setupTest() {
         WebDriverManager.chromedriver().browserVersion("97").setup();
         driver = new ChromeDriver();
+        webDriverWait = new WebDriverWait(driver, Duration.ofSeconds(5));
     }
 
     @AfterTest
-    public void cleasnUpTest() {
+    public void cleanUpTest() {
         driver.close();
         driver.quit();
+
     }
 
     @Test
     public void setDeliveryAreaTest() {
         driver.get("https://www.amazon.com/");
         Assert.assertEquals(driver.getCurrentUrl(), "https://www.amazon.com/",
                 "The current link is not as expected.");
-
         WebElement deliverToIcon = driver.findElement(By.xpath("//*[@id=\"nav-global-location-slot\"]"));
         deliverToIcon.click();
 
-        WebElement zipCodeField = new WebDriverWait(driver, Duration.ofSeconds(5)).until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@id=\"GLUXZipUpdateInput\"]")));
+        WebElement zipCodeField = getUntil(ExpectedConditions.visibilityOfElementLocated(
+                By.xpath("//*[@id=\"GLUXZipUpdateInput\"]")));
         zipCodeField.sendKeys("19901");
-        WebElement applyButton = driver.findElement(By.xpath("//*[@id=\"GLUXZipUpdate\"]/span/input"));
+
+        WebElement applyButton = driver.findElement(
+                By.xpath("//*[@id=\"GLUXZipUpdate\"]/span/input"));
         applyButton.click();
 
         try {
-            WebElement changeButton = new WebDriverWait(driver, Duration.ofSeconds(5)).until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@id=\"nav-main\"]/div[1]/div/div/div[3]/span[2]/span/input")));
+            WebElement changeButton = getUntil(ExpectedConditions.visibilityOfElementLocated(
+                    By.xpath("//*[@id=\"nav-main\"]/div[1]/div/div/div[3]/span[2]/span/input")));
             changeButton.click();
         } catch (RuntimeException e) {
```

Comment:
It is better to specify the error type. I mean If you expect NoElementFoundException or another, instead of RantimeException write the exception what you expect